app/views/numismatic_monograms/index.html.erb

```diff
@@ -12,6 +13,7 @@
       <% @numismatic_monograms.each do |monogram| %>
         <tr>
           <td><%= render_thumbnail_tag monogram, { onerror: default_icon_fallback }, suppress_link: true %></td>
+          <td><%= monogram.decorated_file_sets&.first&.titles&.join(', ') %> </td>
```

Comment:
Do you mind moving the logic 'decorated_file_sets&.first&.titles&.join(', ')' in a method for example 'def decorated_filename' in https://github.com/pulibrary/figgy/blob/master/app/resources/numismatic_monograms/numismatic_monogram_decorator.rb 
and also add a test in https://github.com/pulibrary/figgy/blob/master/spec/resources/numismatic_monograms/numismatic_monogram_decorator_spec.rb